### PR TITLE
[MIG] l10n_ch_states and l10n_ch_zip to v12.0

### PR DESCRIPTION
**Pull request by mpanarin**

### CODE REVIEW

File: l10n_ch_zip/__init__.py [hunk 1]
```diff
@@ -7,6 +7,10 @@ def post_init(cr, registry):
     """Import CSV data as it is faster than xml and because we can't use
     noupdate anymore with csv"""
     from odoo.tools import convert_file
-    filename = 'data/res.better.zip.csv'
-    convert_file(cr, 'l10n_ch_zip', filename, None, mode='init', noupdate=True,
-                 kind='init', report=None)
+    filenames = ['data/res.city.csv', 'data/res.city.zip.csv']
```
Comment (simahawk) [inline l10n_ch_zip/__init__.py]: can you move this code to `hooks.py`?

File: l10n_ch_zip/models/city_zip.py [hunk 2]
```diff
@@ -14,15 +14,28 @@ class ResBetterZip(models.Model):
     http://www.swisspost.ch/post-startseite/post-adress-services-match/post-direct-marketing-datengrundlage/post-match-zip-factsheet.pdf
     """
 
-    _inherit = 'res.better.zip'
+    _inherit = 'res.city.zip'
```
Comment (simahawk) [inline l10n_ch_zip/models/city_zip.py:17]: update class name and docstring

File: l10n_ch_zip/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+* Fill the new field in the partner form with a zip or a city from Switzerland
+* Than you get a list with possible entries.
```
Comment (simahawk) [inline l10n_ch_zip/readme/USAGE.rst]: *then

File: l10n_ch_zip/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+* The one you select is auto-filled in the zip-, citiy-, state- and country-field.
```
Comment (simahawk) [inline l10n_ch_zip/readme/USAGE.rst]: *city

**Review by simahawk — CHANGES_REQUESTED** [review]

File: l10n_ch_zip/__init__.py [hunk 5]
```diff
@@ -7,6 +7,10 @@ def post_init(cr, registry):
     """Import CSV data as it is faster than xml and because we can't use
     noupdate anymore with csv"""
     from odoo.tools import convert_file
```
Comment (sebalix) [inline l10n_ch_zip/__init__.py]: Is it intended to have this import here? Can we move it at the top of the file?

File: l10n_ch_zip/models/city_zip.py [hunk 6]
```diff
@@ -14,15 +14,28 @@ class ResBetterZip(models.Model):
...
-    additional_digit = fields.Char(string='Additional poscode digits', size=2)
+    active = fields.Boolean(
+        string='Active',
+        default=True,
+    )
+    onrp = fields.Char(
+        string='Swiss Post classification no. (ONRP)',
+        size=5,
+        help="Primary Key",
+    )
```
Comment (sebalix) [inline l10n_ch_zip/models/city_zip.py:27]: I don't how this field is used later, do you think it deserves an index?

Comment (yvaucher) [reply l10n_ch_zip/models/city_zip.py:27]: IIRC, this field is there to synchronize the data from swisspost data. So it is not for a daily usage but for once in a while we update the data.

**Pull request merged**